test/parallel_api/iterator/permutation_iterator_common.h

```diff
@@ -67,7 +67,7 @@ wait_and_throw(ExecutionPolicy&& exec)
     struct TestClassName : TestUtils::test_base<TestValueType>
 
 // DEFINE_TEST_PERM_IT_CONSTRUCTOR should be used to declare permutation iterator tests constructor
-#define DEFINE_TEST_PERM_IT_CONSTRUCTOR(TestClassName, ScaleStepValue, ScaleMaxNValue)                                                                 \
+#define DEFINE_TEST_PERM_IT_CONSTRUCTOR(TestClassName, ScaleStepValue, ScaleMaxNValue)                                 \
```

Comment:
These changes seem to be formatting changes unrelated to the PR.  I'd happily approve them separately, but they probably don't belong here.  If you want to make this PR rebased down to 2 or 3 commits, with these separated, that would be find with me.

Reply:
reverted.